### PR TITLE
Fix for viewhelpermanager

### PR DESCRIPTION
**Pull request by masarliev**
In ZF3 name is ViewHelperManager

### CODE REVIEW

**Comment by neeckeloo** [comment]
I encountered the same issue.

**Comment by neeckeloo** [comment]
@markushausammann Could you merge that as soon as possible please ?

File: Module.php [hunk 1]
```diff
@@ -209,7 +209,7 @@ protected function setupExceptionLogging(MvcEvent $event)
      */
     protected function setupJavascriptLogging(MvcEvent $event)
     {
-        $viewHelper = $event->getApplication()->getServiceManager()->get('viewhelpermanager')->get('headscript');
+        $viewHelper = $event->getApplication()->getServiceManager()->get('ViewHelperManager')->get('headscript');
```
Comment (markushausammann) [inline Module.php]: @masarliev @neeckeloo any defining knowledge on this one?

**Comment by markushausammann** [comment]
Can you please make this pull request against the zf3-compat branch.

**Comment by markushausammann** [comment]
As I said before, please pull the `zf3-compat` branch and submit a pull request against this one. Please solve/test the question @neeckeloo raises before you submit the pull request.

**Pull request closed without merging**